fvm/fvm_test.go

```diff
@@ -3065,6 +3065,7 @@ func TestEVM(t *testing.T) {
 		withBootstrapProcedureOptions(fvm.WithSetupEVMEnabled(true)).
```

Comment:
are the changes in this file expected?

Reply:
I'm not sure if they're expected, but I didn't find a way around them given the inclusion of `NonFungibleToken` and `FungibleToken` in the EVM contract. What are the outsized implications of changing this file?

Reply:
I see we derive the addresses from the chain ID. looks good.